tj2-reco.py

```diff
@@ -532,20 +512,22 @@ def str2bool(v):
   parser.add_argument('--datapath', dest='datapath', default='/home/benjamin/desy_tb/', type=str, help='Path to data')
   parser.add_argument('--runno', dest='runno', type=int, help='Run number')
   parser.add_argument('--caltag', dest='caltag', default='', type=str, help='Name of calibration tag to use')
+  parser.add_argument('--prefix', dest='prefix', default='', type=str, help='Name of calibration tag prefix to use')
```

Comment:
prefix is useful when turning on the clusterDB feature -> see useClusterDB flag at beginning of the file